src/commands/go-test.yaml

```diff
@@ -19,6 +19,8 @@ steps:
       echo -n " -X '<< parameters.pkg >>.version=$(cat .project_version)'" >> .ldflags
   - run: |
       echo "\"$(cat .ldflags)\""
+  - run: |
+      go get golang.org/x/tools/cmd/goimports && [[ -n $(goimports -local github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} -l .) ]] && goimports -local github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} -d . && exit 1
```

Comment:
Would be cool if we could start using the name/command tuple here as explained in https://github.com/giantswarm/fmt/blob/master/circleci/all-circleci-best-practices.md

Reply:
Where does the schema live? CI fails because `name` is not defined in the schema. Or did I do something wrong?

```
Error: ERROR IN CONFIG FILE:
[#/commands/go-test] 0 subschemas matched instead of one
1. [#/commands/go-test] expected type: String, found: Mapping
|   Command may be a string reference to another command
2. [#/commands/go-test/steps/8] 0 subschemas matched instead of one
|   1. [#/commands/go-test/steps/8] expected type: String, found: Mapping
|   |   Shorthand commands, like `checkout`
|   |   SCHEMA:
|   |     type: string
|   |   INPUT:
|   |     name: Check if imports are properly sorted
|   |     run: |
|   |       go get golang.org/x/tools/cmd/goimports && [[ -n $(goimports -local github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} -l .) ]] && goimports -local github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} -d . && exit 1
|   2. [#/commands/go-test/steps/8] maximum size: [1], found: [2]
|   |   long form commands like `run:`
|   |   SCHEMA:
|   |     maxProperties: 1
|   |   INPUT:
|   |     name: Check if imports are properly sorted
|   |     run: |
|   |       go get golang.org/x/tools/cmd/goimports && [[ -n $(goimports -local github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} -l .) ]] && goimports -local github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} -d . && exit 1
```

Reply:
It seems circleci does not allow a `name` key here. Reverting this now..